lib/shared/pe_status_check.rb

```diff
@@ -5,6 +5,8 @@
 
 # PEStatusCheck - Shared code for pe_status_check facts
 module PEStatusCheck
+  PUP_PATHS = { server_bin: '/opt/puppetlabs/server/bin' }.freeze
```

Comment:
This is failing as the upper case variable is a constant, that is set multiple times every time the module is called.
I'm not 💯 on the fix without testing, but declaring the variable outside the module, or changing the constant to a lower case variable, or statically coding the bin dir as it's only used once

Reply:
@sandrajiang the rest of the tests are failing just because of the above, it'll probably work when you remove this CONSTANT being set multiple times.

Reply:
This seems weird because the initializations are coming from two different places: the file under `/etc/puppetlabs`:

```
/etc/puppetlabs/code/environments/production/modules/pe_status_check/lib/shared/pe_status_check.rb:8: warning: already initialized constant PEStatusCheck::PUP_PATHS
```

And the cache:

```
/opt/puppetlabs/puppet/cache/lib/shared/pe_status_check.rb:8: warning: previous definition of PUP_PATHS was here
```

I don't know what exactly that means, but we can probably get rid of the warning by changing the line to:

```
PUP_PATHS ||= { server_bin: '/opt/puppetlabs/server/bin' }
```

That will only initialize the constant once.

Reply:
Thanks for the review @m0dular , @MartyEwings. Sorry I missed it, will do it tomorrow.